scripts/index_d_ts

```diff
@@ -2095,6 +2097,7 @@ declare module '@sendbird/uikit-react/ui/MessageInput' {
     onCancelEdit?: () => void,
     onStartTyping?: () => void,
     channelUrl: string,
+    channel: OpenChannel | GroupChannel | null,
```

Comment:
isnt `channel?: OpenChannel | GroupChannel` better

Reply:
Yeah it's in your change. It's been applied too :)